pkg/controller/storagecluster/reconcile.go

```diff
@@ -595,6 +602,62 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 								"--osd-ids=${FAILED_OSD_IDS}",
 								"--namespace=" + sc.Namespace,
 							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "ceph-conf-emptydir",
+									MountPath: "/etc/ceph",
+								},
+							},
+							Env: []corev1.EnvVar{
```

Comment:
Is `POD_NAMESPACE` not needed? It's in the rook upstream example.

Reply:
Updated...